apps/common/models/file-upload.js

```diff
@@ -23,49 +23,14 @@ module.exports = class UploadModel extends Model {
         }
       };
       reqConf.method = 'POST';
+      // uses 'request' function available in HOF model. auth() is used by this process and
+      // thus needs to be declared in the parent class which extends off the HOF model too.
       this.request(reqConf, (err, data) => {
         if (err) {
           return reject(err);
         }
         return resolve(data);
       });
-    }).then(data => {
-      return this.auth().then(bearer => {
-        data.url = (data.url.replace('/file', '/vault')) + '&token=' + bearer.bearer;
```

Comment:
We don't augment any upload url with the token bearer here anymore. We leave it right to then end when the user clicks continue on the Declaration page which is exactly when the payload is submitted to icasework. This means we both generate a token bearer, add it to all the urls and pass it on to iCW without any delay by the user. This means we can reduce the JWT token expiry from 1-2 hours, to literally a few minutes in the Keycloak realm. Also means any failure to get the token in the first place will trigger an error for the user before anything submits, and means we know no latency on their part will cause an expiry issue and thus the duplication problems we've seen on iCW. It should sort all of it out.